selftests/unit/test_virsh.py

```
@@ -1,33 +1,38 @@
-            shell=True,
-        )
-        == 0
-    )
+def virsh_session_is_alive(session_id):
+    """Return whether one particular aexpect-backed virsh session is alive."""
+    from virttest import virsh
+
+    try:
+        session = virsh.VirshSession(a_id=session_id, check_libvirtd=False)
+        return session.is_alive()
+    except Exception:  # The session no longer exists or cannot be attached.
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)